vmdb/db/migrate/20150224192716_migrate_configuration_manager_to_ems.rb

```diff
@@ -0,0 +1,67 @@
+class MigrateConfigurationManagerToEms < ActiveRecord::Migration
+  class ExtManagementSystem < ActiveRecord::Base
+    include UuidMixin
+    self.inheritance_column = :_type_disabled
+  end
+
+  class ConfigurationManager < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class ConfiguredSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class ConfigurationProfile < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    say_with_time("Migrating configuration_managers to ext_management_systems") do
+      systems  = ConfiguredSystem.all.group_by(&:configuration_manager_id)
```

Comment:
Only 3-4 devs will have this populated. I guess it doesn't hurt, but it also has tests and all.